certification/pyxis/types.go

```diff
@@ -1,43 +1,48 @@
 package pyxis
 
-//todo-adam reduce these fields to just what we need for now
 type CertImage struct {
-	Certified              bool          `json:"certified" default:"false"` //todo-adam I'm going to assume we want to set this to false??
-	Deleted                bool          `json:"deleted" default:"false"`
-	DockerImageDigest      string        `json:"docker_image_digest,omitempty"`
-	DockerImageID          string        `json:"docker_image_id,omitempty"`
-	ImageID                string        `json:"image_id,omitempty"`
-	ISVPID                 string        `json:"isv_pid,omitempty"`
-	ParsedData             *ParsedData   `json:"parsed_data,omitempty"`
-	RawConfig              string        `json:"raw_config,omitempty"`
-	Repositories           *Repositories `json:"repositories,omitempty"`
-	SumLayerSizeBytes      int32         `json:"sum_layer_size_bytes,omitempty"`
-	UncompressedTopLayerId string        `json:"uncompressed_top_layer_id,omitempty"`
+	Certified              bool         `json:"certified" default:"false"` //TODO: confirm they want this value sent as false
+	Deleted                bool         `json:"deleted" default:"false"`
+	DockerImageDigest      string       `json:"docker_image_digest,omitempty"`
+	DockerImageID          string       `json:"docker_image_id,omitempty"`
+	ImageID                string       `json:"image_id,omitempty"`
+	ISVPID                 string       `json:"isv_pid,omitempty"`
+	ParsedData             *ParsedData  `json:"parsed_data,omitempty"`
+	RawConfig              string       `json:"raw_config,omitempty"`
+	Repositories           []Repository `json:"repositories,omitempty"`
+	SumLayerSizeBytes      int64        `json:"sum_layer_size_bytes,omitempty"`
+	UncompressedTopLayerId string       `json:"uncompressed_top_layer_id,omitempty"` //todo-adam check if this is required
```

Comment:
It's not a hard enforcement by Pyxis, but we do need it.

Reply:
Yeah, I double checked the spec, the `TODO's` in types file are now more generic, and anyone can find them.